README.md

```diff
@@ -2,7 +2,12 @@
 
 An extremely fast Python type checker and language server, written in Rust.
 
-**This project is still in development and is not ready for production use.**
+> [!WARNING]
+> ty is pre-release software and not ready for production use. Expect to encounter bugs, missing
+> features, and fatal errors.
+
+ty is in active development, and we're working hard to make it stable
+and feature-complete.
```

Comment:
Do you intend to append this? If not, ignore this.
```suggestion
> features, and fatal errors.
> 
> ty is in active development, and we're working hard to make it stable
and feature-complete.
```

Reply:
I did this intentionally -- I didn't feel like this last sentence was part of the "warning", more additional commentary? But it could be wrong, if you think it's better to put it all in the warning box, I can easily do that.

Reply:
No strong preference! 👍